docs/modules/ROOT/pages/index.adoc

```diff
@@ -251,6 +254,68 @@ NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issue
 [#describe-actor-role]
 NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/26[issue #26]): Describe credential holder’s role in relation to the asset.
 
+
+=== Security Considerations for Creating Identity Assertions
+
+Identity Assertions primarily allow a _<<_credential_holder,credential holder>>_ to 
+form a non-repudiable, tamper-evident binding between the _<<_credential_subject,credential subject>>_ and the list of _<<_referenced_assertions,referenced assertions>>._  In some cases, the identity assertion provides an additional trust signal over the information provided by the C2PA Claim Signature, and in other cases, the Identity Assertion is of primary interest, and the Claim Signature provides supplemental information.  
+
+The Identity Assertion straightforwardly supports the former (trust-signal) case, 
+because the Claim Signature integrity protects the identity assertion from stripping or replacement.  This means that the identity assertion cannot be tampered without invalidating the Claim Signature.
+
+However, the converse is not true. Without additional protections, the identity assertion 
+does not guard against the Claim Signature being stripped and replaced.  The mechanisms
+described in this section provide protections against this attack.
+
+The Claim Signature protects against stripping or modification of an identity assertion 
+because the hash of the identity assertion is included in the Claim Signature.  Unfortunately, it is not possible for the identity assertion to include the hash of the 
+Claim Signature because it is not known when the identity assertion is created.  This 
+specification provides a weaker binding that allows the identity assertion creator to 
+specify the key/certificate that is authorized to sign the Claim. This does not prevent a 
+Claim Signature being stripped and replaced by the _same_ Claim Signer, but it does prevent
+the Claim Signature from being replaced by a different Claim Signer.  To enable 
+this protection, the identity assertion creator can optionally include the hash of
+the Claim Signer credential/certificate authorized to (subsequently) sign the Claim.  If these
+protections are not needed (for example, if the identity assertion is expected to be
+a supplemental trust signal), then the `claim_certificate_hash` field can be omitted.
+
+Similar security considerations apply in the case of multiple identity  assertions.  If more than one identity assertion is present, then earlier-created identity assertions that 
```

Comment:
Might be one for @scouten-adobe or @paulengland, what is the use case for signing other identity signatures? Are we intending to convey: "I, Charlie, agree to all the assertions that my colleague, John Smith, is making in his signature"?

Reply:
Yes. 

More broadly - 

An identity assertion is basically the same as a Claim - it signs other assertions.  If a relying party is primarily concerned with the ID assertion signature (i.e., answering the question "who published this") then we'd like the publisher to be able to integrity protect as much as she can so that it can't be stripped and replaced.

Of course, the Claim Signature does this too, but it's easy (and natural) to for the ID signer to sign all previously created ID assertions.  (Arguably, it would be weird not to.)